### PR TITLE
Kocheshkov lambda.part1.exercises 1-3

### PR DESCRIPTION
**Pull request by Mattyou777**

### CODE REVIEW

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  }
+
+  @Test
+  public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator() {
+    Person[] persons = getPersons();
+
+    Arrays.sort(persons, new Comparator<Person>() {
+      @Override
+      public int compare(Person left, Person right) {
+        int LastNameCompare = left.getLastName().compareTo(right.getLastName());
```
Comment (shaart) [inline src/test/java/lambda/part1/exercise/Exercise1.java]: Variable name from lower case

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        new Person("Николай", "Зимов", 30),
+        new Person("Иван", "Мельников", 20)
+    }, persons);
+  }
+
+  @Test
+  public void findFirstWithAge30UsingGuavaPredicate() {
+    List<Person> persons = Arrays.asList(getPersons());
+
+    Predicate<Person> isAge30Checker = new Predicate<Person>() {
```
Comment (shaart) [inline src/test/java/lambda/part1/exercise/Exercise1.java]: Here must be declared class which implements Predicate and it's instance, not anonymous class

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 1 above)
Comment (Mattyou777) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: Thanks.
Fixed.

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 2 above)
Comment (Mattyou777) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: Thanks.
Fixed.

**Pull request merged**